airmozilla/popcorn/static/popcorn/js/editor.js

```diff
@@ -27,11 +27,7 @@ $(document).ready(function () {
             csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val()
         })
         .done(function (response) {
-            var message = "Your edit has been saved and sent it for "
-            + "transcoding. Once it completes, which can take several minutes, "
-            + "we'll automatically update the event to use your latest "
-            + "video edits.";
-            alert(message);
+            document.location = '/popcorn/status/' + $('#editor').data('slug');
```

Comment:
Due to the way Popcorn works, I think we should put an alert here of our own. 
Also, what can we do to prevent Popcorn throwing up that "Leave page?" dialog? How about we use jQUery to destroy the iframe. Do you think that'd work?